spec/datadog/ci/contrib/cucumber/instrumentation_spec.rb

```diff
@@ -202,14 +203,45 @@
     end
   end
 
+  context "executing a scenario with examples" do
+    let(:feature_file_to_run) { "with_parameters.feature" }
+
+    it "a single test suite, and a test span for each example with parameters JSON" do
+      expect(test_spans.count).to eq(3)
+      expect(test_suite_spans.count).to eq(1)
+
+      test_spans.each_with_index do |span, index|
+        # test parameters are available since cucumber 4
+        if cucumber_4_or_above
+          expect(span.get_tag(Datadog::CI::Ext::Test::TAG_NAME)).to eq("scenario with examples")
+
+          expect(span.get_tag(Datadog::CI::Ext::Test::TAG_PARAMETERS)).to eq(
+            "{\"num1\":\"#{index}\",\"num2\":\"#{index + 1}\",\"total\":\"#{index + index + 1}\"}"
```

Comment:
does the UI render these parameters correctly? The format the other tracers follow is 
```
{
  "arguments": // ... your parameters
  "metadata": { // ... extra metadata }
}
```
<img width="301" alt="Screenshot 2024-01-11 at 13 03 33" src="https://github.com/DataDog/datadog-ci-rb/assets/22798219/15b2f751-9ce1-4dcb-a249-ce61c05cd34d">

Reply:
Working on it, UI is handling it correctly:

<img width="753" alt="image" src="https://github.com/DataDog/datadog-ci-rb/assets/426400/b1bd2bd7-3149-452c-9c8b-6d666405ba93">

Reply:
@juan-fernandez fixed:
<img width="1209" alt="image" src="https://github.com/DataDog/datadog-ci-rb/assets/426400/f10936b3-4f53-4ed5-888e-4f5216a5b918">